draft-ietf-taps-interface.md

```diff
@@ -3556,6 +3556,8 @@ in {{security-parameters}}. It does not recommend use (or disuse) of specific
 algorithms or protocols. Any API-compatible transport security protocol ought to work in a Transport Services system.
 Security considerations for these protocols are discussed in the respective specifications.
 
+{{I-D.ietf-taps-arch}} outlines general security considerations and requirements for any system that implements the Transport Services architecture. These include recommendations of relevance to the API, e.g. regarding the use of keying material.
```

Comment:
/outlines/provides/ ? (i didn't much like "outlines `...` requirements"

Reply:
ok by me (I don't see the problem, but I'm not a native speaker). This applies to the -impl draft as well, I'll propose a fix to both.